openmc/universe.py

```diff
@@ -701,13 +705,24 @@ def get_unique_universes(self):
 
         """
 
-        unique_universes = np.unique(self._universes.ravel())
-        universes = {}
-
-        for universe in unique_universes:
-            universes[universe._id] = universe
-
-        return universes
+        univs = dict()
```

Comment:
Wow that sucks. I can't believe they didn't keep a routine like ``flatten`` around.